crates/optimism/primitives/src/transaction/signed.rs

```diff
@@ -0,0 +1,440 @@
+//! A signed Optimism transaction.
+
+use core::mem;
+
```

Comment:
```suggestion
```

Reply:
this keeps standard/no_std deps separate from other deps so it's easy to identify them